features/step_definitions/job.rb

```diff
@@ -4,3 +4,17 @@
 
   raise "job #{job.name} never completed" unless @result[:success]
 end
+
+Given /^a job is present with labels:$/ do |table|
```

Comment:
`a job appears with labels`, when using `present` it sounds like we will not wait for it. I know we have 1 or 2 steps with such names but that was wrong.